benchmark/bm_entangling_layers.py

```diff
@@ -0,0 +1,43 @@
+"""Entangling layers benchmark.
+Creates an immutable QNode using the StronglyEntanglingLayers template,
+then evaluates it and its Jacobian.
+"""
+# pylint: disable=invalid-name
+import numpy as np
+
+import pennylane as qml
+from pennylane.templates.layers import StronglyEntanglingLayers
+from pennylane.templates.embeddings import AngleEmbedding
+from pennylane.init import strong_ent_layers_uniform
+
+n = 3
+n_min = 1
+n_max = 5
+n_step = 1
+
+n_wires = 3
+
+dev = qml.device("default.qubit", wires=n_wires)
```

Comment:
Would be great if benchmarks could be portable across different devices. Then we could easily compare speeds

Reply:
Good idea

Reply:
Now the user can specify a list of devices to run the benchmark on.